### ISSUE TITLE
Problem with binder version of Investigating star formation in the W5 region 

### ISSUE THREAD

**Issue by hcferguson** [glue-viz/glue-jupyter#60]
Seems like the notebook on binder is maybe missing an import? Cell 4:

``` scatter_viewer = app.scatter2d('[4.5]-[5.8]', '[8.0]', data=data_catalog) ``` 

fails with the following traceback:

```
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-4-6ae31bc350b4> in <module>
----> 1 scatter_viewer = app.scatter2d('[4.5]-[5.8]', '[8.0]', data=data_catalog)

/srv/conda/lib/python3.6/site-packages/glue_jupyter/app.py in scatter2d(self, x, y, data, widget, color, size, viewer_state, layer_state)
    183     def scatter2d(self, x=None, y=None, data=None, widget='bqplot', color=None, size=None, viewer_state=None, layer_state=None):
    184         if widget == 'bqplot':
--> 185             from .bqplot import BqplotScatterView
    186             viewer_cls = BqplotScatterView
    187         elif widget == 'matplotlib':

/srv/conda/lib/python3.6/site-packages/glue_jupyter/bqplot/__init__.py in <module>
----> 1 from .view import BqplotScatterView, BqplotImageView, BqplotHistogramView

/srv/conda/lib/python3.6/site-packages/glue_jupyter/bqplot/view.py in <module>
    241 from glue.viewers.image.composite_array import CompositeArray
    242 
--> 243 from .image import BqplotImageLayerArtist
    244 from .scatter import BqplotScatterLayerArtist
    245 from ..utils import rgba_to_png_data

/srv/conda/lib/python3.6/site-packages/glue_jupyter/bqplot/image.py in <module>
      3 import numpy as np
      4 import bqplot
----> 5 from ipyastroimage.astroimage import AstroImage
      6 import ipywidgets as widgets
      7 import ipywidgets.widgets.trait_types as tt

ModuleNotFoundError: No module named 'ipyastroimage'
```

**Comment by astrofrog** [comment]
Yeah a lot of the functionality is currently broken on binder as there are some new dependencies that need to be set up there. I'll have it all fixed shortly.

**Comment by astrofrog** [comment]
@hcferguson - I think this should now be fixed. Can you try:

https://mybinder.org/v2/gh/glue-viz/glue-jupyter/master?filepath=notebooks%2FAstronomy%2FW5%2FW5%20Tutorial.ipynb

and let me know if not?

**Comment by hcferguson** [comment]
That fixed the problem in that I can run the whole notebook.

In running this on Safari, hit two new issues:

- The interactivity is very laggy
  - In one attempt, the first plot didn't show up after more than 60 seconds; in other attempts it took more than 5 seconds for the selection to show up. Presumably this is the downside of using Binder for this.
- The selections in two dimensions aren't selecting the right points (screenshot attached; the selection box is the little shaded box. The points selected are colored red).

<img width="1080" alt="screen shot 2019-02-21 at 7 03 40 pm" src="https://user-images.githubusercontent.com/2261329/53210338-752e0080-360b-11e9-8fa3-397afdaf68be.png">

**Comment by astrofrog** [comment]
@hcferguson - 60 seconds does seem very slow. Indeed it could be that at peak times binder is just quite slow. Hopefully it'll be easy to test out locally once we have conda dev packages for this.

For the second issue, I noticed that yesterday too and opened a bug report: https://github.com/glue-viz/glue-jupyter/issues/65

**Comment by astrofrog** [comment]
I'm going to close this as the selection bug is tracked in #65 (and fixed in #69) and the lagging is just due to the performance of binder, so I'm not sure if there's much we can do about it (however feel free to open a new issue if this persists)
